api/sourcegraph.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * use Vercel Serverless Function to proxy sourcegraph API
+ * @author netcon
+ */
+
+const httpProxy = require('http-proxy');
+
+// just proxy the request to sourcegraph directly
+const proxy = httpProxy.createProxyServer({
```

Comment:
Is this for bypassing CORS? Should we add a comment about it?

Reply:
Yes, this is used for bypassing CORS, I am trying to contact sourcegraph about it, now it is just for a taste